sympy/vector/parametricregion.py

```diff
@@ -0,0 +1,48 @@
+from sympy.core.basic import Basic
+
+class ParametricRegion(Basic):
+    """
+    Represents a parametric region in space.
+    """
+    def __init__(self, system, *args):
+        """
+        Examples
+        ========
+
+        >>> from sympy import symbols, cos, sin, pi
+        >>> from sympy.vector import CoordSys3D, ParametricRegion
+        >>> r, theta = symbols("r theta")
+        >>> C = CoordSys3D('C')
+        >>> circle = ParametricRegion(C, r*cos(theta), r*sin(theta), (theta, 0, 2*pi))
+        """
+        self._definition = list()
+        self._bounds = list()
+        self._parameters = list()
+        self.system = system
+
+        for arg in args:
+            if isinstance(arg, tuple):
+                self.bounds.append(arg)
+            else:
+                self.definition.append(arg)
+
+        if len(self.definition) > 3:
+            raise ValueError("Only 3-D regions are supported")
+
+        if len(self.bounds) > 3:
+            raise ValueError("Only 3-D regions are supported")
+
+        for elem in self.bounds:
+            self.parameters.append(elem[0])
+
+    @property
+    def definition(self):
+        return self._definition
+
+    @property
+    def bounds(self):
+        return self._bounds
```

Comment:
Maybe we should return bounds in the form of dict().
```
{t: (0, 1)}
```
This form will be more useful for 'ParametricIntegral` class.

Reply:
Pay attention that dictionaries are not ordered. Maybe we care about the order of parameters?

Reply:
OK. It is better to use a list or tuple.